[package]
name = "boost/throw_exception"
version = "1.63.0"
edition = "2011"
authors = ["Ken Matsui <26405363+ken-matsui@users.noreply.github.com>"]
repository = "https://github.com/boostorg/throw_exception/tree/boost-1.63.0"
documentation = "http://boost.org/libs/throw_exception"
readme = "README.md"
description = """
Boost.IO
"""

[dependencies]
"boost/assert" = "1.66.0"
"boost/config" = "1.66.0"
